experimental/dds/tree2/src/test/simple-tree/list.spec.ts

```diff
@@ -560,64 +560,64 @@ describe("List", () => {
 				check(["a", "b"], "a", /* start: */ -Infinity);
 			});
 
-			describe("at()", () => {
-				const check = (array: readonly string[], index: unknown) => {
-					test2("at", array, noInit, index);
-				};
-
-				// "Normal" indices
-				check([], 0);
-				check([], -0);
-				check([], -1);
-				check(["a", "b"], 0);
-				check(["a", "b"], 1);
-				check(["a", "b"], 2);
-				check(["a", "b"], -1);
-				check(["a", "b"], -2);
-				check(["a", "b"], -3);
-				// Non-integer indices
-				check(["a", "b"], 0.5);
-				check(["a", "b"], 1.5);
-				check(["a", "b"], -0.5); // Truncated to 0 - first element
-				check(["a", "b"], -1.5); // Truncated to -1 - second element
-				check(["a", "b"], -2.5); // Truncated to -2 - first element
-				// Non-integer indices at and close to the valid "edges"
-				check(["a", "b"], 1.999999); // Truncated to -1 - second element
-				check(["a", "b"], 2.0); // Truncated to -2 - first element
-				check(["a", "b"], 2.000001); // Truncated to -2 - first element
-				check(["a", "b"], -2.999999); // Truncated to -2 - first element
-				check(["a", "b"], -3.0); // Truncated to -3 - out of bounds
-				check(["a", "b"], -3.000001); // Truncated to -3 - out of bounds
-				check(["a", "b"], -3.5); // Truncated to -3 - out of bounds
-				// Extreme values
-				check(["a", "b"], Infinity);
-				check(["a", "b"], -Infinity);
-				check(["a", "b"], Number.MAX_SAFE_INTEGER);
-				check(["a", "b"], Number.MIN_SAFE_INTEGER);
-				check(["a", "b"], Number.MAX_VALUE);
-				check(["a", "b"], Number.MIN_VALUE);
-				check(["a", "b"], Number.EPSILON);
-				// Indices that are not numbers
-				check(["a", "b"], "0");
-				check(["a", "b"], "1");
-				check(["a", "b"], "1.999999");
-				check(["a", "b"], "2.0");
-				check(["a", "b"], "-0");
-				check(["a", "b"], "-1");
-				check(["a", "b"], "-2.999999");
-				check(["a", "b"], "-3.0");
-				check(["a", "b"], "not-a-number");
-				check(["a", "b"], NaN);
-				check(["a", "b"], true);
-				check(["a", "b"], false);
-				check(["a", "b"], undefined);
-				check(["a", "b"], null);
-				check(["a", "b"], {});
-				check(["a", "b"], { a: 1, b: 2 });
-				// TODO: validate these throw; need to update the test harness to support checking that both versions throw
-				// check(["a", "b"], Symbol("MySymbol"));
-				// check(["a", "b"], BigInt(1));
-			});
+			// describe("at()", () => {
```

Comment:
We're uncommenting these before merging, right?

Reply:
I'm in the process of reverting and unreverting changes in this PR to binary search for something that's causing tests to hang (not a single test, so it's not an infinite loop because otherwise it would timeout. the entire test run hangs and fails without giving a reason). Hence the several force pushes. This will be cleaned up before merge